src/map/conquest_data.h

```diff
@@ -0,0 +1,68 @@
+#ifndef SERVER_CONQUEST_DATA_H
+#define SERVER_CONQUEST_DATA_H
+
+#include <string>
+
+#include "common/cbasetypes.h"
+#include "conquest_zmq.h"
+#include "zone.h"
+
+class ConquestData
+{
+public:
+    ConquestData();
+
+    /**
+     * Loads the latest conquest data from DB.
+     * Should only ever be done on map initialization.
+     * World server should be updating conquest data periodically.
+     */
+    void load(std::unique_ptr<SqlConnection>& sql);
+
+    /**
+     * Gets the influence points for a given nation in a given region.
+     */
+    int32 getInfluence(REGION_TYPE region, uint8 nation);
```

Comment:
Can we start upgrading nation to its own enum like REGION_TYPE?

Reply:
Done

Reply:
To be cleared created the enum and used it in conquest, but wont refactor outside